lib/todo.rb

```diff
@@ -20,9 +20,14 @@ def contents
     end #contents
 
     def get_contents
+      database.rewind
```

Comment:
What's the reasoning here for adding the rewind instead of using the memoization added to the `contents` method from #11 ?  If change your calls to `get_contents` in `display` and `is_empty` to use `contents` instead of `get_contents` this should be un-needed and result in only one IO read.